homeassistant/components/telegram_bot/__init__.py

```diff
@@ -67,6 +67,7 @@
 ATTR_USERNAME = "username"
 ATTR_VERIFY_SSL = "verify_ssl"
 ATTR_TIMEOUT = "timeout"
+ATTR_INTERNAL_REQUEST_ID = "internal_request_id"
```

Comment:
It took me a bit to figure out that this was intended to tag the message so you could figure out the id when it is sent.

How about calling this `message_tag` instead?

Reply:
This is a shorter and more understandable name.

Reply:
Renamed